hawc/apps/common/validators.py

```diff
@@ -113,6 +113,8 @@ def validate_hyperlink(value: str, raise_exception: bool = True) -> bool:
     valid = url.scheme in valid_scheme and (
         url.netloc == "" or any(url.netloc.endswith(ending) for ending in valid_netloc_endings)
     )
+    if url.netloc == "" and ("." in url.path or not url.path.endswith("/")):
```

Comment:
What's the reasoning for disallowing filename extensions? At the very least should we allow for `.html`?

Reply:
There's no valid URL on the hawc website that doesn't end with a trailing slash or a filename extension. Many times security bots try to add paths like this based on old php applications that weren't secured well and would allow arbitrary execution or retrieval of files on disk.